### PR TITLE
Issue337 - Fixed reset issue

### LINKED ISSUE #337: Reset Search Field (Request-Customer, Request-Admin, Appointment-Admin)
Reset button for 
- [ ] Request (Customer), 
- [ ] Request (Admin),
- [x] Appointment(Admin)

Reset button for Request (Customer), Request (Admin),
[Before]
image
[After]
image

- [search by day] and the [search value] has been reset
    - [start-date] has not been reset
    - [end-date] has not been reset
    - [search type] has not been reset

Reset button for Appointment(Admin)

[Before]
image

[After]
image

- [search by day] and the [search by specific date] has been reset
    - [search type] has not been reset
    - [search value] has not been reset

### PR DESCRIPTION
**Pull request by sonechca**
Can reset all of field

### CODE REVIEW

**Review by yuanLeeMidori — COMMENTED** [review]
I tried these,

1. Select one date, search, reset (works well so far)
2. Select customer search type and input emma, click search (nothing happen)

I check the code an I think it goes to the previous else if block in `update()` (see my other comment)

File: src/Appointment/Admin/AppointmentsAdmin.js [hunk 1]
```diff
@@ -66,29 +66,29 @@ class AppointmentsAdmin extends React.Component {
...
   };
 
   updateAppointment() {
     if (this.state.searchDate) {
       const newAppointments = this.state.appointments.filter((req) => {
-        return moment(req.schedule.date.date).isSame(this.state.searchDate); 
+        return moment(req.schedule.date.date).isSame(this.state.searchDate);
       });
       this.setState({ filterAppointments: newAppointments });
     } else if (this.state.dayValue) {
```
Comment (yuanLeeMidori) [inline src/Appointment/Admin/AppointmentsAdmin.js:80]: If we change it to
`else if (this.state.dayValue != 9999) {`
it'll solve the issue I face (see my other comment)

Please test it and see if this is a solution, thanks!

**Review by eunbeek — COMMENTED** [review]
value={this.state.searchType}

Could you add this value for search Type reset ?
I did not test that working well after adding this value in form

**Comment by sonechca** [comment]
value={this.state.searchType}

is already set in the searchType. 
I just added searchType: data in the reset function
It worked well

**Comment by yuanLeeMidori** [comment]
@sonechca fix the conflict and rebase it so our review won't be dismissed :)

**Comment by sonechca** [comment]
Fixed conflict :)

**Review by yuanLeeMidori — APPROVED** [review]

**Review by rko7 — APPROVED** [review]

**Pull request merged**